workspaces/ui-v2/src/store/documentationEdit/thunks.ts

```diff
@@ -27,6 +28,21 @@ const fetchRemoveEndpointCommands = async (
   }
 };
 
+const fetchFieldEditCommands = async (
+  spectacle: IForkableSpectacle,
+  fieldId: string,
+  requestedTypes: JsonType[]
+): Promise<CQRSCommand[]> => {
+  const spectacleClient = new SpectacleClient(spectacle);
+  try {
+    return spectacleClient.fetchFieldEditCommands(fieldId, requestedTypes);
+  } catch (e) {
+    console.error(e);
+    Sentry.captureException(e);
```

Comment:
We don't have a Sentry handler anywhere further at the "edge" of our program, forwarding all thrown errors, do we? Seems weird to see vendor specific stuff so deeply integrated, but I guess it's pragmatic?

Reply:
Hmmm, we don't want the errors to be propagated past the redux store, but there is probably some middleware we can listen to rejected promises and sentry error, rather than add this to each thunk. I'll look into this and put up a separate PR